Daytime-Operations/Safety/Control-Safety-Systems.rst

```diff
@@ -0,0 +1,112 @@
+Performing LOTO is also required if you are accessing an area where there is a collision risk.
+
+**Lockout:** This method uses physical locks to stop anyone from turning equipment back on. 
+If you use LOTO, you must have your own unique lock issued to you by the Safety or SIT-Com team. 
+Once you apply your lock to a piece of equipment, only you can take it off. 
+You are responsible for knowing the procedure to properly den-energize and lockout the equipment. 
+
+**Tagout:** If you place a lock on a piece of equipment, you must also include a tag to indicate that the machine or piece of equipment is not usable and being serviced. 
+These tags are available around the summit, or can be issued to you for you to write your contact information on. 
+
+LOTO needs to occur anytime you put yourself or a team near any part of the dangerous zone of a piece of equipment. 
+Some examples are:
+
+    -	When accessing the observing platform on the TMA, EVEN IF the power supply is off
+    -	Going up to the Main Dome aperture platform to open the shutters
+    -	Doing any physical work on the AuxTel mount, such as adding accelerometers or installing sensors
+
+You are required to complete the `LOTO training <https://drive.google.com/file/d/1zKtvqbjtLcOuefIXvZ4KTSdCNThznWs9/view?usp=share_link>`_ before you are issued a lock and tag. 
+This video is a overview of the OSHA guidelines for performing LOTO.
+It does not include specific procedures for performing LOTO on any Rubin Observatory equipment. 
+Once you have reviewed the video and available LOTO procedures, register your compliance with the Safety team and sign the register they maintain on the summit, affirming that you have completed the required training. 
+
+.. _Daytime-Operations-Safety-Control-Safety-Systems-GIS:
+
+GIS
+^^^
+The Global Interlock System (GIS) is a safety system that guarantees the Rubin Observatory's secure operation. 
+GIS interacts with other systems of the Simonyi and Auxiliary Telescopes, such as the Pflow, M1M3, M2 Hexapod, etc. 
+If any of the interlocks are triggered, such as fire detection, seismic activity, or a safety gate is open, such as the safety gate on the 6th floor underneath the TMA azimuth wrap, GIS receives these signals and creates a waterfall effect that blocks off any system coupled to the trigger. 
+This is an example of the order of operations the GIS manages when an interlock is triggered:
+
+1.	The TMA is rotating in azimuth.
+2.	There is an earthquake or seismic activity detected by the Observatory’s earthquake monitoring system in the EAS (Environmental Awareness System).
+3.	GIS receives these signals. 
+4.	GIS sends a detection signal.
+5.	The TMA is blocked from rotating, and the CSCs go into error state, and the EUI displays a safety interlock.
+
+GIS is a reactionary system, and is not fit for LOTO given that it does not completely remove an energy source from a system, it only temporarily blocks the energy source by software stop. 
```

Comment:
"by a software stop" --> is that true? Aren't buttons required to be manually pushed in order to reset it?

Reply:
A touchscreen can be used to release it, or there is a button on the safety gate, I guess. 👍